.github/workflows/release.yaml

```diff
@@ -395,6 +395,17 @@ jobs:
           prerelease: false
           draft: true
           fail_on_unmatched_files: true
+      - name: Checkout index.yaml from main branch
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          ref: main
+          sparse-checkout: config/helm/repos/stable/index.yaml
+          sparse-checkout-cone-mode: false
```

Comment:
"non-cone mode is deprecated" => https://git-scm.com/docs/git-sparse-checkout#_internalsnon_cone_problems

Reply:
I’m not sure if it’s a good idea to introduce deprecated features, even though the GitHub Actions documentation mentions them.

I would prefer to do this without using an action:

```
git restore --source main -- config/helm/repos/stable/index.yaml
```

This way, you can also skip the “Copy index.yaml file” step.

Reply:
Awesome idea! It works :+1: I just updated the PR